.github/workflows/_linux_ut.yml

```diff
@@ -142,3 +144,31 @@ jobs:
             test_cmd="${test_cmd} test_xpu.py"
           fi
           eval $test_cmd
```

Comment:
How about inductor test logs? will we upload those logs too? Seems the pytorch ut test log under `./test/test-reports/` by default